auth-server/src/types.ts

```diff
@@ -258,3 +254,29 @@ export enum PostTypeEnum {
 }
 
 export type PostType = PostTypeEnum;
+
+export interface AddressSignupStartArgs {
+    address: string;
+    network: string;
+}
+
+export interface AddressSignupConfirmArgs {
```

Comment:
We should have only, `address`, `network` and `signature`

Reply:
done